temporian/core/operators/BUILD

```diff
@@ -449,3 +449,16 @@ py_library(
         "//temporian/proto:core_py_proto",
     ],
 )
+
+py_library(
+    name = "drop",
+    srcs = ["drop.py"],
+    srcs_version = "PY3",
+    deps = [
+        ":base",
+        "//temporian/core:operator_lib",
+        "//temporian/core/data:node",
+        "//temporian/core/data:schema",
+        "//temporian/proto:core_py_proto",
+    ],
```

Comment:
these two don't seem necessary: 
```
"//temporian/core/data:schema",
"//temporian/proto:core_py_proto",
```

and these seem missing:
```
"//temporian/core:compilation",
"//temporian/core:typing",
"//temporian/core/operators:select",
```

try running the `python tools/build_cleaner.py` (though its weird these tests passed with missing deps)